src/components/calls-page/header-actions.tsx

```diff
@@ -36,26 +37,43 @@ const MuteAllCallsBtn = styled(PrimaryButton)`
 
 const HeaderButtons = styled.div`
   display: flex;
+  gap: 1rem;
 `;
 
 type HeaderActionsProps = {
   isEmpty: boolean;
   isSingleCall: boolean;
   isMasterInputMuted: boolean;
-  setIsMasterInputMuted: (isMasterInputMuted: boolean) => void;
   addCallActive: boolean;
+  isConnected: boolean;
+  isReconnecting: boolean;
+  setIsMasterInputMuted: (isMasterInputMuted: boolean) => void;
   setAddCallActive: (addCallActive: boolean) => void;
+  connect: (url: string) => void;
+  disconnect: () => void;
```

Comment:
A minor thing for clarity: could be confusion with plain-"prop-names" like isConnected,  isReconnecting, connect, disconnect when we are in components which has webRTC-connection as its main function and these props have nothing to do with the call-connection. 

So maybe add WS to it or something like that.